Library/lua/LuaScript.cpp

```diff
@@ -25,16 +26,8 @@ bool LuaScript::Initialize(const std::wstring& scriptFile)
 {
 	assert(!IsInitialized());
 
-	FILE* file = _wfopen(scriptFile.c_str(), L"rb");
-	if (!file) return false;
-
-	fseek(file, 0, SEEK_END);
-	const long fileSize = ftell(file);
-	BYTE* fileData = new BYTE[fileSize];
-	fseek(file, 0, SEEK_SET);
-	fread(fileData, fileSize, 1, file);
-	fclose(file);
-	file = nullptr;
+	size_t fileSize = 0;
+	auto fileData = FileUtil::ReadFullFile(scriptFile, &fileSize);

```

Comment:
We should check if `fileData` is valid (like `if (!file) return false;` in the removed code).